swan-accpy/Dockerfile

```diff
@@ -30,6 +30,9 @@ RUN dnf install -y \
     dnf clean all && \
     rm -rf /var/cache/dnf
 
+# Install UV package manager for Python (faster that pip)
+RUN HOME=/home/${NB_USER} curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
This runs as user root, so root will be the owner of these files. And it seems you are creating something in the home directory of the user, who should own those files, not root.